wdl/transforms/draft3/src/test/scala/wdl/draft3/transforms/wdlom2wom/WdlFileToWomSpec.scala

```diff
@@ -57,6 +57,7 @@ class WdlFileToWomSpec extends FlatSpec with Matchers {
     "input_values" -> anyWomWillDo,
     "passthrough_workflow" -> anyWomWillDo,
     "scatter_var_member_access" -> anyWomWillDo,
+    "nested_conditionals" -> anyWomWillDo,
```

Comment:
If all you want to do is check that a workflow will validate correctly, I'd consider putting this as a valid`WomtoolValidateSpec` case